mitxpro/urls.py

```diff
@@ -76,6 +76,7 @@
     # social django needs to be here to preempt the login
     path("", include("social_django.urls", namespace="social")),
     re_path(r"^dashboard/", index, name="user-dashboard"),
+    re_path(r"^receipt/", index, name="order-receipt"),
```

Comment:
it's giving empty page. As per my understanding it should be loaded only with some id like `receipt/order_id`.

Reply:
Updated